internal/server/v1/triples/golden/bulk_triples_in/result1.json

```diff
@@ -10366,5 +10366,5 @@
       "node": "dummy"
     }
   ],
-  "nextToken": "H4sIAAAAAAAA/2zOwQrCMAyAYZt1WxY8jAji86h3hYEnQeqIUljb0XUHfWofQVavXj5C/kNCV+K9DUN42t4M3Si9deKpSq9RTo9/jVcMqBgQGLBgQM2AJQNWrLFul4IM2DAg0ZvWhzD7dDtLnIKnrRMzzVGc+HQUH5z1JoVImy6ZZKe0HLqYaM19EIZWsUaV/c1FVmfLbNUqLrHefQrWiHnV5B/oCwAA//8BAAD//0LbYVbaAAAA"
+  "nextToken": "H4sIAAAAAAAA/2zOwQrCMAzGcZd1WxY8jAg+kHpXGHgSpI4ohbUdXXfY28vi1cuPwP+Qjx7EJxfH+HGDHftJBuclUJ3XSa7vf413DFgwIDBgyYCGASsGrNlg020FGbBlQKKV9ue4hPy8SZpjoKMXOy9JvIR8kRC9CzbHRIc+2+zmvD262+TsaxSGrmCDhfq7S9WolVqrjYpqqxPoCwAA//8BAAD//yvpAtjZAAAA"
```

Comment:
For this one diff, can you use this token in the test.go file https://github.com/datacommonsorg/mixer/blob/ab802f7c91e3be0341caa69d72cd4b9ff8191125/internal/server/v1/triples/golden/bulk_triples_in_test.go#L51 and similar for the one of internal/server/v2/propertyvalues/golden/triple_in/result1.json

Reply:
Done